### ISSUE TITLE
Support ddoc as description for helptexts

### ISSUE THREAD

**Issue by gizmomogwai** [andrey-zherikov/argparse#156]
I was playing with rust/clap and one feature I really liked there was, that you can use doc-comments as an alternative to annotations to document flags/commands...
Not sure that its possible to get to the doc-comments in dlang though.

**Comment by andrey-zherikov** [comment]
I think I got what you are looking for but to be sure, could you please provide more details (e.g., an example) of what you want?

**Comment by gizmomogwai** [comment]
instead of 
```
    @(NamedArgument(["aFlag"]).Description("test description"))
    bool aFlag;
```
it would perhaps be interesting to be able to write
```
    /// test description
    @(NamedArgument(["aFlag"]))
    bool aFlag;
```

if both variants are there, then the explicit description should win.
Unfortunately I think there is no way to get to the `///` information at compile time.

**Comment by gizmomogwai** [comment]
Might be something for opend ... perhaps this feature will be "back" ported to dlang :)

**Comment by andrey-zherikov** [comment]
Yeah. I'm not sure it's implemented already but I'll definitely take a look.
